### PR TITLE
Added support for the next release (3.4.0)

### PR DESCRIPTION
**Pull request by jonasraoni**

### CODE REVIEW

File: DefaultTranslationPlugin.inc.php [hunk 1]
```diff
@@ -3,86 +3,75 @@
 /**
  * @file DefaultTranslationPlugin.inc.php
  *
- * Copyright (c) 2013-2020 Simon Fraser University
- * Copyright (c) 2003-2020 John Willinsky
+ * Copyright (c) 2013-2022 Simon Fraser University
+ * Copyright (c) 2003-2022 John Willinsky
```
Comment (jonasraoni) [inline DefaultTranslationPlugin.inc.php:7]: I always forget to ask about these copyright dates, isn't it feasible to adopt a format like "Copyright (c) 2015 - Present XXXX"?

**Review by jonasraoni — COMMENTED** [review]
Reminder: The changes will require the creation of a new branch for stable-3_3_0.

File: DefaultTranslationPlugin.inc.php (hunk 1 above)
Comment (asmecher) [reply DefaultTranslationPlugin.inc.php]: To be honest I'm not sure, and have just kept updating the dates for consistency's sake. If you can find an example of a major project that does this, I'd be happy to adopt.

**Comment by asmecher** [comment]
Looks great, @jonasraoni -- nice work adapting to modern PHP.

File: DefaultTranslationPlugin.inc.php (hunk 1 above)
Comment (jonasraoni) [reply DefaultTranslationPlugin.inc.php]: Ok! I've created an issue with what I found here: https://github.com/pkp/pkp-lib/issues/7818

**Pull request merged**